include/effectmanager.h

```diff
@@ -329,6 +329,18 @@ class  EffectManager : public IJSONSerializable
 
         CRGB oldColor = lastManualColor;
         lastManualColor = color;
+        
+        //This block shifts the hue of the received color to be used in the audio reactive monochromatic fire effect.
+        //Becaue of scope errors by the compiler, the color conversion happens here.
+        CHSV colorVarientHSV = rgb2hsv_approximate(color);

```

Comment:
I will be glad to fix the formatting. I really just wanted to explain my work for the pull request and can remove the comments completely. Although someone might read the code later and wonder why that code block is in the wrong place. 

A varient is when a person intends to write "variant" but, is bad at spelling and writes the same misspelling over and over. This is an embarrassing mistake. Will fix.

As to the placement of the color shift block: It bugs me that I had to put it there. It made no sense as to why it kept failing to compile. I should have captured the error to be more precise. So, just now, I put the code where it makes sense in order to se what the error was. It compiled just fine. I recently rebooted the computer. Maybe that did the trick. No offense meant towards gcc. The problem probably lies in the complex relationship between VS Code, platform IO, whatever plugins are installed, and the fact that my Mac probably is not happy with all of the MSSoft software installed.

Wrapping around the hue: I will trust you if you say it will safely wrap around. I prefer to test it before I full send, but I don't have the correct hardware. I picked up some microphones to wire to my board. But, since I'm using a Heltec Wi-FI Kit V3 and not the m5stick, there are many, many errors thrown at compile time when enabling audio and setting the input pin. Anyway, the worst that will happen without warping is that if a color has a hue of 255, the variant will just be the same color. Truly monochromatic.

Fresh commit incoming.